packages/web/pages/user/my-account/curate-technology/[id].js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import { Protected } from '../../../../components/Authorization';
+import { UserProfile } from '../../../../components/UserProfile';
+import { getTechnologiesToCurate } from '../../../../services';
+import { ROLES as rolesEnum } from '../../../../utils/enums/api.enum';
+import CurateTechnology from '../../../../components/CurateTechnology';
+import EmptyScreen from '../../../../components/EmptyScreen';
+
+const CurateTechnologyPage = ({ technology }) => {
+	return (
+		<Container>
+			<Protected userRole={rolesEnum.REVIEWER}>
+				<UserProfile />
+				<MainContentContainer>
+					{technology ? (
+						<CurateTechnology technology={technology} />
+					) : (
+						<EmptyScreen message="Não existem dados a serem exibidos para essa tecnologia" />
+					)}
+				</MainContentContainer>
+			</Protected>
+		</Container>
+	);
+};
+
+CurateTechnologyPage.getInitialProps = async (ctx) => {
+	const { query } = ctx;
+
+	const { technologies = [] } =
+		(await getTechnologiesToCurate({
+			ids: query.id,
+		})) || {};
```

Comment:
Shouldn't the fallback be an empty array?
And shouldn't that be in the service return statement?
```suggestion
		await getTechnologiesToCurate({
			ids: query.id,
		});
```

Reply:
Yeah you're right that's redundant